src/physics.js

```diff
@@ -0,0 +1,81 @@
+import AABB from './aabb';
+import QuadTree from './qtree';
+import Victor from 'victor';
+
+export default class Physics {
+    constructor(levelBoundArr, timeStep = 10, gravity = 9.82) {
+        this.physicsBodyArr = new Array();
+        this.timeStepFactor = timeStep/1000;
+        this.gravity = gravity; //units/(s^2)
+        this.quadTree = new QuadTree(0, 0, levelBoundArr[0], levelBoundArr[1]);
+    }
+
+    addBody(physicsBody) {
+        physicsBody.quadID = this.physicsBodyArr.length;
+        this.physicsBodyArr.push(physicsBody);
+        this.quadTree.put(physicsBody);
+    }
+
+    resolveArea(viewPortPhysicsBody) {
+        for (const physicsBody of this.physicsBodyArr) {
+            physicsBody.inView = this.collides(physicsBody, viewPortPhysicsBody);
+            this.resolveBody(physicsBody);
```

Comment:
right now I am wondering how costly it would be to simulate all bodies in a level.